tests/integration/transaction_freeze_e2e_test.py

```
@@ -0,0 +1,146 @@
+    tx.freeze_with(executor_client) 
+    tx.sign(executor_key)
+    receipt = tx.execute(executor_client)
+
+    # Verify that the transaction_bodys are generated for the provided node_account_ids only
+    assert len(tx._transaction_body_bytes) == 2
+    assert set(tx._transaction_body_bytes.keys()) == set(node_account_ids)
+
+    assert receipt.status == ResponseCode.SUCCESS, "Transaction must execute successfully"
+
+@pytest.mark.integration
+def test_transaction_executes_successfully_with_single_node_account_id(env):
```

Ruff (0.14.10)

Redefinition of unused `env` from line 9

(F811)